src/app/components/details/details.css

```diff
@@ -1,44 +0,0 @@
-label.details__album-label {
```

Comment:
I prefer that we keep these local css files to make it easier to maintain styles if you don't mind

Reply:
Some of these styles will just be deleted and some will be moved to app.css, like the song list styling, because it's used across multiple components.

Reply:
but the playlist details thingy is definitely specific to this component :/